dht.go

```diff
@@ -144,11 +144,13 @@ func NewDHTClient(ctx context.Context, h host.Host, dstore ds.Batching) *IpfsDHT
 }
 
 func makeDHT(ctx context.Context, h host.Host, dstore ds.Batching, protocols []protocol.ID, bucketSize int) *IpfsDHT {
-	rt := kb.NewRoutingTable(bucketSize, kb.ConvertPeerID(h.ID()), time.Minute, h.Peerstore())
+	self := kb.ConvertPeerID(h.ID())
+	rt := kb.NewRoutingTable(bucketSize, self, time.Minute, h.Peerstore())
 	cmgr := h.ConnManager()
 
 	rt.PeerAdded = func(p peer.ID) {
-		cmgr.TagPeer(p, "kbucket", 5)
+		dist := kb.CommonPrefixLen(self, kb.ConvertPeerID(p))
```

Comment:
let's not call this `dist` -- it's precisely the opposite, i.e. common equal prefix bits. The higher the dist(ance), the lesser common prefix bits you have.

And we want to boost the score the closer we are to the peer.

Reply:
Thanks! I knew dad, but brain fart haha